wire-ios/Wire-iOS/Sources/UserInterface/UserProfile/Devices/ProfileClientViewController.swift

```diff
@@ -502,10 +506,6 @@ extension ProfileClientViewController: UserClientObserver {
 
     func userClientDidChange(_ changeInfo: UserClientChangeInfo) {
 
-        if changeInfo.fingerprintChanged {
-            updateFingerprintLabel()
-        }
```

Comment:
right now, no. Does it translate as we get a new UserClient?

Reply:
I think it would be this scenario:

1. User A adds a new client (A2) but no message is sent so no session exists
2. You navigate the the device list of user A and look at A2

I suppose we establish a session with A2 at upon navigating to the client details screen but if this is async I suppose the fingerprint will not be visible initially.